src/main/java/ch/njol/skript/expressions/ExprMiddleOfLocation.java

```diff
@@ -0,0 +1,60 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import org.bukkit.Location;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Middle of Location")
+@Description("Returns the Middle/Center of a location. In other words returns the middle of the X and Z coords of a location.")
```

Comment:
```suggestion
@Description("Returns the middle/center of a location. In other words returns the middle of the X and Z coordinates of a location.")
```
This description does not describe this expression enough, for example it doesn't mention what happens to the Y coordinate

Reply:
Sure, I will edit that and add more info about Y coordinate